_dasldemos/proof-troubleshooting.md

```diff
@@ -0,0 +1,116 @@
+
+### Identify why the task failed
+
+- If you scroll down even further, you’ll notice a “Job Failures” table. Click “Get/Refresh Failed Job Metadata” to populate the table with details about your failed task. The `failures.causedBy.message` columns are of particular interest as they contain the error messages produced by the tool used within the task itself. In this example pictured below, I provided an erroneous path for one of my inputs, and as you can see, the tool “could not localize” it:
+
+![job_failures](/dasldemos/assets/proof_ts_job_failures.png)
+
+- If these messages don’t provide enough information, navigate back to the “Job List” table and take a look at a few additional columns:
+    - **stdout & stderr**: these columns contain the location of flat files that contain the output and error messages from the command line tools used during the task(s) in question. These will help to provide tool-specific feedback as to what might be going wrong. To access these, copy the path provided, log on to Rhino, and view the text files via `vi` , `vim`, `cat` , or some other similar text viewing tool.
+    - **commandLine**: this column contains the exact command that was run during this task. Check to see if anything looks off, maybe an improper string concatenation, maybe a reference to the wrong WDL variable. If so, navigate to the “command” section of the task to see what might be causing the issue.
+    - **docker**: this column contains the name of the Docker container that was used in the task. First, check for typos, but if the image name and version tag appear to be correct, try pulling the container locally using Docker (or Apptainer on Rhino/Gizmo). If it pulls successfully, try running exactly what is written in the commandLine column within the container.
+
+## Execution Issues
+
+### Check the workflow metadata
+
+- If these resources still don’t provide any sort of illumination, go back to the “Workflow Run” table and copy the workflow ID number via the “copyId” column. Navigate to the [“Troubleshoot" tab of PROOF](/dasldemos/proof-how-to#troubleshooting), paste the workflow ID into the “Troubleshoot a Workflow” section, and click “Get Complete Workflow Metadata”. This should display a large unformatted json, mostly containing the same information as the tables mentioned above, but it could contain a detail that was missed by PROOF’s user interface.
+- Along the same lines, you can also look into the raw workflow log produced by your underlying PROOF server. To do this, identify your workflow log directory on the “PROOF Server” tab (see screenshot below) and navigate to that directory on Rhino. Once there, you will see a collection of files following the naming convention of `workflow.[workflowid].log`. Open the logfile corresponding to the workflow in question using your text viewer of choice (vi, vim, cat, VS Code) to see details about the Cromwell execution of your WDL script.
```

Comment:
Again, I'd suggest more and less instead of vi and vim.

Reply:
Great call, updated this to just say "`more`, `less`, `cat`, VS Code".